tests/integration_tests_control_files/isentropic_vortex_integration/2D_inviscid_isentropic_vortex_h_refinement.prm

```diff
@@ -0,0 +1,61 @@
+# Listing of Parameters
+# ---------------------
+# Number of dimensions
+
+set dimension = 2 
+set run_type = integration_test
+set test_type = h_refinement_study_isentropic_vortex
+set pde_type = euler
+
+# DG formulation
+set use_weak_form = false
+set use_split_form = true
+set use_collocated_nodes = true 
+
+set flux_reconstruction = cDG
+set use_inverse_mass_on_the_fly = true
+
+# Note: this was added to turn off check_same_coords() -- has no other function when dim!=1
+set use_periodic_bc = true
+
+# numerical fluxes
+set conv_num_flux = two_point_flux
+set two_point_num_flux_type = IR
+
+# ODE solver
+subsection ODE solver
+  set ode_output = quiet 
+  set ode_solver_type = runge_kutta 
+  #set ode_solver_type = rrk_explicit # currently not available
+  set output_solution_every_x_steps = -1
+  set output_solution_every_dt_time_intervals = 100.0 
+  set runge_kutta_method = rk4_ex
+end
+
+subsection linear solver
+  set linear_solver_output = quiet
+end
+
+# freestream Mach number
+subsection euler
+  set mach_infinity = 1.195228609334 #=sqrt(2/1.4)
+end
+
+subsection time_refinement_study
+  set number_of_times_to_solve = 5
+  set refinement_ratio = 0.5
+end
+
+subsection flow_solver
+  set flow_case_type = isentropic_vortex
+  set poly_degree = 3
+  set final_time = 23.66431913239 #=20 * sqrt(1.4), time for vortex to advect around the domain
+  set courant_friedrich_lewy_number = 0.1
```

Comment:
Minor point but should be spelled courant–friedrichs–lewy.

Reply:
That's a very good point, I don't know how we have missed that! I'll change it as part of this PR.